addons/website/controllers/form.py

```diff
@@ -296,7 +320,7 @@ def insert_record(self, request, model_sudo, values, custom, meta=None):
         return record.id
 
     # Link all files attached on the form
-    def insert_attachment(self, model_sudo, id_record, files):
+    def insert_attachment(self, model_sudo, id_record, files, template=None):
```

Comment:
Maybe name the `template` parameter `template_sudo` to avoid accidentally using it to access unauthorized model records. (also in the enterprise PR)